### ISSUE TITLE
'verbs'?

### ISSUE THREAD

**Issue by RosieRedfield** [STAT545-UBC/Discussion#341]
Why did Jenny tell us to title today's work 'dplyr single table verbs'?

**Comment by ksedivyhaley** [comment]
Guess: because they perform an action (verb) on a single table / data frame?

arrange() arranges your table, group_by() groups your table.

**Comment by jennybc** [comment]
Yeah, I think I'm just used to hearing this set of functions described as "verbs": things you can do to or with a single data frame. `filter()`, `select()`, `mutate()`, `rename()`, `arrange()`, `summarize()`, etc.

**Comment by jennybc** [comment]
This is actual `dplyr` lingo: it occurs near the top of this vignette:

https://cran.rstudio.com/web/packages/dplyr/vignettes/introduction.html

and I have fallen into the same habit.

**Comment by RosieRedfield** [comment]
I found what looks to be WIckham's original Tidy article.  There 'verb' starts out as more of a metaphor, but already starts morphing into jargon.
